config/main.py

```diff
@@ -2166,6 +2166,29 @@ def delete(ctx):
     sflow_tbl['global'].pop('agent_id')
     config_db.set_entry('SFLOW', 'global', sflow_tbl['global'])
 
+#
+# 'feature' command ('config feature name state')
+# 
+@config.command('feature')
+@click.argument('name', metavar='<feature-name>', required=True)
+@click.argument('state', metavar='<enabled/disabled>', default='disabled', required=True, type=click.Choice(["enabled", "disabled"]))
+def feature_status(name,state):
+    """ Configure status of feature"""
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    status_data = config_db.get_entry('FEATURE', name)
+
+    if not name:
+        click.echo("Invalid feature name")
+        return
+    if not status_data:
```

Comment:
why does it matter and is considered as a failure case?

Reply:
The thought here, I believe, is that if status_data is None, there was no key named "<name>" in the FEATURES table, therefore <name> is not a valid feature to modify.

Reply:
That's correct